bakerydemo/base/management/commands/reset_admin_password.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib.auth.models import User
```

Comment:
suggestion: implement https://docs.wagtail.org/en/stable/reference/hooks.html#before-edit-user to prevent the password change in the first place?

Reply:
I originally did that, but using signals, and it ended up a little brittle. I'll give it another whirl

Reply:
hmm.. I don't think this will work.

Reply:
Ended up being more complex than either of us thought, especially as usernames can change too: https://github.com/wagtail/bakerydemo/pull/369/commits/7b74ab114e58f0d85f189b629500a925be28211c